guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java

```diff
@@ -26,7 +26,8 @@
 import com.google.gwt.user.client.ui.PopupPanel;
```

Comment:
@manstis Not yet. I will make this changes in others files if this one rewrited correctly. Just want to know is it correct changes and it's full list of files needs changes.